webroot/results/admin/export/template.README.txt

```diff
@@ -1,103 +1,135 @@
+| Competitions                            | WCA competitions                                   |
+| Events                                  | WCA events (3x3x3 Cube, Megaminx, etc)             |
+| Results                                 | WCA results per competition+event+round+person     |
+| RanksSingle                             | Best single result per competitor+event and ranks  |
+| RanksAverage                            | Best average result per competitor+event and ranks |
+| Rounds                                  | The round types (first, final, etc)                |
+| Formats                                 | The round formats (best of 3, average of 5, etc)   |
+| Countries                               | Countries                                          |
+| Continents                              | Continents                                         |
+| Scrambles                               | Scrambles                                          |
+| championships                           | Championship competitions                          |
+| eligible_country_iso2s_for_championship | See explanation below                              |
+
+Most of the tables should be self-explanatory, but here are a few specific details:
+
+### Countries
 
 Countries stores include those from the Wikipedia list of countries at
 http://en.wikipedia.org/wiki/List_of_countries, and may include some countries
-that no longer exist.  The ISO2 column should reflect ISO 3166-1 alpha-2
-country codes, for countries that have them.  Custom codes may used in some
+that no longer exist. The ISO2 column should reflect ISO 3166-1 alpha-2
+country codes, for countries that have them. Custom codes may used in some
 circumstances.
 
-eligible_country_iso2s_for_championship stores information about which citizenships are eligible
-to win special cross-country championship types. For example, "greater_china" is a special
-championship type which contains 4 iso2: CN, HK, MC and TW. This means competitors from China,
-Hong Kong, Macau, and Taiwan are eligible to win a competition with championship type "greater_china".
-
-Scrambles stores all scrambles. For 333mbf, an attempt is comprised of multiple, newline separated
-scrambles. Unfortunately, newlines don't work so well in TSV, so we use "|" as a scramble delimiter
-instead.
-
-Result values of the Results table can be interpreted as follows:
-- The result values are in fields value1-value5, best and average.
-- Value -1 means DNF
-- Value -2 means DNS
-- Value 0 means "nothing", for example a best-of-3 has value4=value5=average=0
-- Positive values depend on the event, see column "format" in Events.
-  - Most events have format "time", where the value represents centiseconds.
+### Scrambles
+
+Scrambles stores all scrambles.
+
+For the `333mbf` event, an attempt is comprised of multiple scrambles. The
+scrambles for an attempt are delimited using the `|` character instead of
```

Comment:
nit: This technically changes the meaning. This is worth double checking, but I believe that currently, our CSV export splits scrambles by newlines, and our TSV export splits scrambles by `|` (yes, that's quite a silly incompatibility).

Reply:
Edit, whoops, we don't have a CSV export, ignore that comment. I meant SQL. See https://github.com/thewca/worldcubeassociation.org/pull/2169.